src/Enum/ValueTypeEnum.php

```diff
@@ -38,4 +38,23 @@ final class ValueTypeEnum
      * @var string
      */
     public const TYPE_UUID = 'uuid';
+
+    /**
+     * {@inheritDoc}
+     *
+     * @api
+     *
+     * @return array<string>
+     */
+    public static function getAllValueTypes(): array
```

Comment:
we don't need it in contracts

Reply:
Or split this class static::TYPE_PATH,  static::TYPE_UUID, should be SettingValueTypeEnum extends ValueTypeEnum

Reply:
@pavelmaksimov25 We can do it now or in the next epic